userguide/services.rst

```diff
@@ -134,57 +135,41 @@ which are described in
-   | Home Share Time Machine | checkbox       | when checked, enables Time Machine Lock Stealing; Apple recommends that shares designated for Time Machine      |
-   |                         |                | backups be used exclusively for Time Machine backups                                                            |
+   | Max. Connections        | integer        | Maximum number of simultaneous connections.                                                                     |
    |                         |                |                                                                                                                 |
    +-------------------------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Database Path           | browse button  | select the path to store the CNID databases used by AFP (default is the root of the pool); the path must be     |
-   |                         |                | writable                                                                                                        |
+   | Database Path           | browse button  | Select the path to store the CNID databases used by AFP (default is the root of the pool). The path must be     |
+   |                         |                | writable.                                                                                                       |
    +-------------------------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Global auxiliary        | string         | additional `afp.conf(5) <http://netatalk.sourceforge.net/3.0/htmldocs/afp.conf.5.html>`_                        |
-   | parameters              |                | parameters not covered elsewhere in this screen                                                                 |
+   | Chmod Request           | drop-down menu | Indicates how to handle ACLs. choices are *Ignore*, *Preserve*, or *Simple*.                                    |
    |                         |                |                                                                                                                 |
    +-------------------------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Map ACLs                | drop-down menu | choose mapping of effective permissions for authenticated users; *Rights* (default, Unix-style permissions),    |
-   |                         |                | *Mode* (ACLs), or *None*                                                                                        |
+   | Map ACLs                | drop-down menu | Choose mapping of effective permissions for authenticated users. *Rights* (default, Unix-style permissions),    |
```

Comment:
Combine sentences so choices are part of the first sentence: "users: *Rights*..."